server/svix-server/src/worker.rs

```diff
@@ -401,10 +421,18 @@ async fn dispatch(
         Err((attempt, err)) => {
             let attempt = attempt.insert(db).await?;
 
+            let retry_schedule_override =
+                get_retry_schedule_override(db, cache, org_id, event_type_name).await?;
+
+            let retry_schedule = match retry_schedule_override {
+                Some(retry_schedule) => retry_schedule.to_durations(),
+                None => cfg.retry_schedule.to_owned(),
+            };
```

Comment:
`to_owned()` is essentially a clone() here. It'll also copy the data...

Just do something like:
```rust
let retry_schedule = retry_schedule_override.map(|x| x.to_durations()).unwrap_or(cfg.retry_schedule);
```

Reply:
Yeah, I know to_owned is often just a clone. That code above does not work either, because cfg.retry_schedule is behind an Arc and does not implement Copy. Although I do like it in that its terser than a match haha.

Reply:
We don't want `&Option<Vec<Duration>>`, we want `&Vec<Duration>`. The problem is that `Vet<Duration>` is created inside the `to_durations()` method. The compiler suggests binding it with `let` so that it lives longer, but we'd have to unwrap it from the Option first -- which is when we fallback on `cfg.retry_schedule`.

Reply:
I had a small mistake there, here's the full code:
```rust
let retry_schedule_orig = retry_schedule_override.map(|x| x.to_durations());
let retry_schedule = retry_schedule_orig.as_deref().unwrap_or(&cfg.retry_schedule);
```

Reply:
Oh neat. I will remember this for the future.

Reply:
It's quite possible you can even reuse the name (so no need for the _orig nonsense), I just didn't want to potentially have it not work.